tests/TestCase.php

```diff
@@ -24,6 +24,7 @@ abstract class TestCase extends BaseTestCase
 
     protected function setUp(): void
     {
+        Builder::rebuild();
```

Comment:
Why?

Reply:
I was debugging, forgot to remove.Thanks.